app/experimenter/experiments/models/nimbus.py

```diff
@@ -0,0 +1,82 @@
+from django.contrib.auth import get_user_model
+from django.contrib.postgres.fields import ArrayField
+from django.core.validators import MaxValueValidator
+from django.db import models
+
+from experimenter.projects.models import Project
+from experimenter.experiments.constants import ExperimentConstants
```

Comment:
Oh and I was gonna say we should start a new constants file for Nimbus but we can also do that as a followup